packages/SketchMorph2-Tests.package/M2DrawingComponentTests.class/instance/execute..st

```diff
@@ -0,0 +1,14 @@
+support
+execute: aBlockArray
```

Comment:
`execute` doesn't imply testing the undo functionality here. Maybe the name should include the purpose of testing undo.

Reply:
Yeah, you're right. This method was once designated to have more parameters, one of which would have implied the undo action. Since they are not needed anymore, I'll rename the method.